archinfo/arch.py

```diff
@@ -1,3 +1,5 @@
+from builtins import map, chr, object
```

Comment:
Why is this necessary?

Reply:
Well, that's what `futurize` recommend to add to as in here http://python-future.org/imports.html. Let me remove it and retry in Python 3